doc/modules/cross_validation.rst

```diff
@@ -290,7 +290,7 @@ The function :func:`cross_val_predict` is appropriate for:
 The available cross validation iterators are introduced in the following
 section.
 
-.. topic:: Examples
+.. dropdown:: Examples
```

Comment:
Can we fold bigger things, rather than these very small folds, to remove more crowing.